types/map-stream/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for map-stream 0.0
+// Project: http://github.com/dominictarr/map-stream
+// Definitions by: Konrad Perlicki <https://github.com/KonradPerlicki>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = mapStream;
+
+declare namespace mapStream {
+    interface Options {
+        /**
+         * Continue mapping even if error occured.
+         *
+         * On error `map-stream` will emit `failure` event.
+         *
+         * Default: `false`
+         */
+        failures?: boolean;
+    }
+
+    type Callback = (err: null | Error, data: unknown) => void;
+}
+
+declare function mapStream(
+    mapper: (data: unknown, callback: mapStream.Callback) => unknown,
```

Comment:
does it return unknown or just void (ev undefined):
https://github.com/dominictarr/map-stream/blob/master/index.js#L83-L85

Reply:
It seems to me that it doesn't return anything

Reply:
cool, so just type `void` (so no one is expected to consume return results)